scripts/change_owner.js

```diff
@@ -1,7 +1,7 @@
 require('dotenv').config({path:'./.env'});
 const hre = require('hardhat');
 const {PROVIDER_KEY, MNEMONIC} = process.env;
-const {'4': {SeedFactory: SEED_FACTORY, Safe}} = require('../contract-addresses.json');
+const {'rinkeby': {SeedFactory: SEED_FACTORY, Safe}} = require('../contractAddresses.json');
```

Comment:
let's not hardcode the network name in the script, network name could be stored in the `.env` file, pls reference to the `contracts` repo